### ISSUE TITLE
Mention Fedora Package on main website

### ISSUE THREAD

**Issue by 768kb** [GothenburgBitFactory/timewarrior#449]
You should mention that on Fedora, there is a Package Repository Version of timewarrior called "timew", which can be installed easily with `dnf install timew`.

**Comment by tbabej** [comment]
Good point! Fedora is actually mentioned in our README, and docs should also refer to the package - `timew` has been available there since Fedora 26, after all :slightly_smiling_face:

**Comment by tbabej** [comment]
Moving to the website repo.
